README.md

```diff
@@ -26,7 +26,7 @@ QUICKINSTALL:
 ```
 # cd espressopp
 # cmake -DEXTERNAL_BOOST=OFF -DEXTERNAL_MPI4PY=OFF .
-# make -j -l$(nproc)
+# make -j -l$(nproc)-2
```

Comment:
I am still not convinced of the point of this change, using load=nproc-2 is as arbitrary as load=nproc.

Plus, when `make` parses the option using `atof()`, `atof(x-2)` becomes `atof(x)`, which effectively ignores the `-2`.